agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitMultiProjectPropertiesLocator.java

```diff
@@ -41,16 +42,16 @@ public GitMultiProjectPropertiesLocator(DelayedTeamscaleMultiProjectUploader upl
 	 * Asynchronously searches the given jar file for a git.properties file.
 	 */
 	@Override
-	public void searchJarFileForGitPropertiesAsync(File jarFile) {
-		executor.execute(() -> searchJarFile(jarFile));
+	public void searchJarFileForGitPropertiesAsync(Pair<File, Boolean> file) {
```

Comment:
Below there are quite some logging calls that should now use file.getFirst() instead. WDYT of making this two arguments instead of one pair so that you can give them descriptive names?